apps/meteor/app/mentions/server/getMentionedTeamMembers.ts

```diff
@@ -1,35 +1,27 @@
 import { Team } from '@rocket.chat/core-services';
-import type { IMessage } from '@rocket.chat/core-typings';
 
 import { callbacks } from '../../../lib/callbacks';
 import { settings } from '../../settings/server';
 
-interface IExtraDataForNotification {
-	userMentions: any[];
-	otherMentions: any[];
-	message: IMessage;
+interface ITeamMention {
+	_id: string;
+	name: string;
+	type: string;
```

Comment:
is type just a string?

Reply:
Changed to MessageMention type. https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/24287f1295a72f2347a26fb852804dd18dff9907